File: clean_nav.js
const fs = require('fs');
const path = require('path');

function walk(dir, callback) {
    fs.readdirSync(dir).forEach(f => {
        let dirPath = path.join(dir, f);
        let isDirectory = fs.statSync(dirPath).isDirectory();
        isDirectory ? walk(dirPath, callback) : callback(path.join(dir, f));
    });
}

walk('c:/Users/shrir/shreezatech/src/app', (filepath) => {
    if (filepath.endsWith('.jsx') && !filepath.endsWith('layout.jsx')) {
        let content = fs.readFileSync(filepath, 'utf-8');
        let newContent = content;
        
        // Remove imports
        newContent = newContent.replace(/import\s+Navbar\s+from\s+['"].*?Navbar.*?['"];?\n?/g, '');
        newContent = newContent.replace(/import\s+Footer\s+from\s+['"].*?Footer.*?['"];?\n?/g, '');
        
        // Remove components
        newContent = newContent.replace(/\s*<Navbar\s*\/>\s*/g, '\n');
        newContent = newContent.replace(/\s*<Footer\s*\/>\s*/g, '\n');
        
        if (content !== newContent) {
            fs.writeFileSync(filepath, newContent, 'utf-8');
            console.log('Cleaned ' + filepath);
        }
    }
});

File: clean_nav.py
import os
import re

directory = r'c:\Users\shrir\shreezatech\src\app'
for root, dirs, files in os.walk(directory):
    for file in files:
        if file.endswith('.jsx') and file != 'layout.jsx':
            filepath = os.path.join(root, file)
            with open(filepath, 'r', encoding='utf-8') as f:
                content = f.read()
            
            new_content = content
            # Remove imports
            new_content = re.sub(r'import\s+Navbar\s+from\s+[\'\"].*?Navbar.*?[\'\"];?\n?', '', new_content)
            new_content = re.sub(r'import\s+Footer\s+from\s+[\'\"].*?Footer.*?[\'\"];?\n?', '', new_content)
            
            # Remove components
            new_content = re.sub(r'\s*<Navbar\s*/>\s*', '\n', new_content)
            new_content = re.sub(r'\s*<Footer\s*/>\s*', '\n', new_content)
            
            if content != new_content:
                with open(filepath, 'w', encoding='utf-8') as f:
                    f.write(new_content)
                print(f'Cleaned {filepath}')
